hack/generator/pkg/codegen/pipeline_determine_resource_ownership.go

```diff
@@ -197,15 +198,10 @@ func updateChildResourceDefinitionsWithOwner(
 		// TODO: equality check to find the name of the actual resource, but we can't do that check
 		// TODO: now because these types allOf inherit from resourceBase and the actual resources
 		// TODO: being referenced do not. See: https://github.com/Azure/k8s-infra/issues/211
-		if typeName.Name() == "VirtualMachines_Spec_Resources" || // Uses allof inheritance
-			typeName.Name() == "Account_Spec_Resources" || // Uses allof inheritance
-			typeName.Name() == "Sites_Spec_Resources" || // Uses allof inheritance
-			typeName.Name() == "Namespaces_Spec_Resources" || // Uses allof inheritance
-			typeName.Name() == "Vaults_Spec_Resources" || // Uses allof inheritance
+		if typeName.Name() == "ServersAdministrators" ||
+			typeName.Name() == "ExtensionsChild" {
```

Comment:
Why this change?

Reply:
Right, they didn’t use to end up as `ResourceType`s but now they do.